python/veles/proto/exceptions.py

```diff
@@ -125,3 +126,17 @@ class RegistryMultiMatchError(VelesException):
 class ConnectionLostError(VelesException):
     code = 'connection_lost'
     msg = "Connection lost"
+
+
+class CriticalException(VelesException):
```

Comment:
I wanted to add exception sub-type that when thrown will cause connection to be closed.